pulsar-broker/pom.xml

```diff
@@ -230,6 +230,11 @@
       <artifactId>jersey-media-json-jackson</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
Why do we need to specify the dependency here? Pinning the version in the `dependencyManagement` section in top level pom should be enough, no?

Reply:
O it isn't.
The dependencyManagement section is like a default configuration but it deos not mean that dependencies declared up there take precedence